modules/jdbc/src/main/java/org/apache/ignite/internal/jdbc/JdbcPreparedStatement.java

```diff
@@ -762,7 +762,7 @@ private void executeWithArguments(JdbcStatementType statementType) throws SQLExc
         Object[] args = currentArgs == null ? ArrayUtils.OBJECT_EMPTY_ARRAY :
                 currentArgs.stream().map(this::convertJdbcTypeToInternal).toArray();
 
-        execute0(statementType, sql, args);
+        execute0(statementType, sql, false, args);
```

Comment:
I don't know where to put this question.

Why don't we allow script execution using PreparedStatement#execute method?

Reply:
yep, u a right, fixed